core-api/src/main/resources/definitions/exposed-services.json

```diff
@@ -639,6 +639,20 @@
       "apiIncluded": false,
       "visibleForDatalake": false,
       "visibleForDatahub": true
+    },
+    {
+      "name": "KNOX_TOKEN_GENERATOR",
+      "displayName": "Knox Token Generation",
+      "serviceName": "KNOX",
+      "knoxService": "KNOX_TOKEN_GENERATOR",
+      "knoxUrl": "/tokengen/index.html",
```

Comment:
As I mentioned in the slack channel, I think that we want this to be more of a Token Integrations link at this point. This will point to the Knox /homepage/home link which will display the homepage with default settings focused on the cdp-proxy-token topology and related links. It will also have the link to Token Generation page on the home page. This allows us to NOT have to add a whole other set of API endpoints for cdp-proxy-token and the user having to know when to use which.

I know this is different than was described in the JIRA - we only came to this conclusion on Friday. 

I am also curious whether that knoxUrl link works like that without being fully qualified - maybe you fully qualify it internally upon deployment? I don't think where that link lives that it would work as a root relative path from CP.